libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -549,18 +549,11 @@ - (void)setupServiceConfiguration
 - (void)logoutCleanup
 {
     self.passcodeDisplayed = NO;
-    self.screenLockDisplayed = NO;
 }
 
 - (void)handleAppForeground:(NSNotification *)notification
 {
-    [SFSDKCoreLogger d:[self class] format:@"App is entering the foreground."];
-    if (self.isScreenLockDisplayed) {
```

Comment:
This check was carried over from the old passcode system but not needed anymore.